src/library_wasmfs.js

```diff
@@ -0,0 +1,8 @@
+var WASMFS = {
```

Comment:
Do you think we'll be filling out this implementation soon? If so, including this file might be worth it if you add a comment explaining that it is a placeholder for future work with a link to the tracking issue. Otherwise adding this file would probably be best left to when we have something to put in it.